src/main/resources/application.yaml

```diff
@@ -19,4 +19,13 @@ management:
 FAMILIE_BA_MOTTAK_URL: dummy
 PDL_API_URL: dummy
 
+no.nav.security.jwt:
+  issuers: selvbetjening
+  issuer.selvbetjening:
+    discoveryurl: https://login.microsoftonline.com/NAVtestB2C.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1A_idporten_ver1
+    accepted_audience: ${ACCEPTEDAUDIENCE}
+    cookie_name: selvbetjening-idtoken
+
+STS_URL: https://api-gw-q1.oera.no/security-token-service
```

Comment:
Når vi setter miljøvariabler for dev i `application.yaml` kan vi kanskje fjerne det fra `application-dev.yaml` så det ikke blir dobbelt :)

`application-dev.yaml` vil så vidt jeg vet uansett overskrive verdier som er satt fra `application.yaml`, så har ikke noe å si egentlig, men greit med alt som kan fjernes!

Reply:
🧠